In [3]:
import pandas as pd
import numpy as np

In [4]:
df = pd.read_csv('../datos/df_2016.csv')
df.sample(3)
df.shape

(13051, 12)

# Pandas IV

1 - Cargad el csv que guardasteis ayer.
Antes de seguir con las preguntas, para evitar posibles errores con los nulos vamos a eliminar todos los nulos del DataFrame, esto lo haremos sobre el archivo de 2016.xlsx. Para ello vamos a utilizar el método .dropna() de Pandas. 

Tendréis que ejecutar la siguiente línea de código para eliminar los nulos del DataFrame y poder seguir haciendo el resto de los ejercicios.
```
df.dropna(inplace = True)
```
¿Cuántas filas se han eliminado con esta línea de código?

In [5]:
df.dropna(inplace=True)
df.shape

(12901, 12)

In [6]:
df.sample(10)

id_ticket       fecha  id_empleado  id_agente      categoría  \
11035  TDLTNT-9042476552  2016-04-16        355.0       15.0  Acceso&%Login   
1287   GHLESR-2242733296  2016-12-29        669.0        2.0        Sistema   
861    GDLTSR-5842592233  2016-08-10        433.0       12.0        Sistema   
2848   GMLTNR-3142519207  2016-05-29       1470.0       12.0  Acceso&%Login   
6361   KHLTST-8342507626  2016-05-17       1252.0       46.0        Sistema   
8243   SDLTNR-4142621264  2016-09-08       1156.0       32.0  Acceso&%Login   
11603  THLTER-8642659625  2016-10-16       1522.0       46.0       Hardware   
4145   KDLTER-5042680378  2016-11-06       1267.0       13.0       Hardware   
176    GDLESR-2442625785  2016-09-12       1128.0       47.0        Sistema   
6357   KHLTST-7842558197  2016-07-07        839.0       21.0        Sistema   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
11035  Solicitud  2 - Normal  0 - Sin Asignar              0.0           4.0   
1287    Problema  2 - Normal         3 - Alta              2.0           5.0   
861    Solicitud  2 - Normal  0 - Sin Asignar             18.0           5.0   
2848   Solicitud  2 - Normal        2 - Media              1.0           5.0   
6361   Solicitud  2 - Normal         3 - Alta              6.0           4.0   
8243   Solicitud  2 - Normal  0 - Sin Asignar              1.0           5.0   
11603  Solicitud  2 - Normal         3 - Alta              9.0           4.0   
4145   Solicitud  2 - Normal  0 - Sin Asignar             15.0           3.0   
176     Problema  2 - Normal  0 - Sin Asignar              4.0           5.0   
6357   Solicitud  2 - Normal         3 - Alta              8.0           1.0   

                    web  índice_  
11035  www.tractchun.es      1.0  
1287   www.tractchun.es      1.0  
861    www.tractchun.es      1.0  
2848   www.tractchun.es      1.0  
6361   www.tractchun.es      1.0  
8243   www.tractchun.es      1.0  
11603  www.tractchun.es      1.0  
4145   www.tractchun.es      1.0  
176    www.tractchun.es      1.0  
6357   www.tractchun.es      1.0

2 - ¿Cuál es el número de incidencias que se han dado de tipo "Sistema", lo encontraréis en la columna de categoría?

In [7]:
cond1 = df['categoría'] == 'Sistema'
df_sistema = df[cond1]
print(df_sistema.shape)

(5203, 12)


In [8]:
df_sistema.head()
# hay 5203 incidencias del sistema

id_ticket       fecha  id_empleado  id_agente categoría  \
2    GDDESR-1342539995  2016-06-18        569.0       29.0   Sistema   
3    GDDTSR-5942488006  2016-04-28        320.0       40.0   Sistema   
156  GDLESR-0142497241  2016-05-07        584.0        2.0   Sistema   
157  GDLESR-0142558772  2016-07-07       1147.0       47.0   Sistema   
158  GDLESR-0242517083  2016-05-27        578.0       10.0   Sistema   

          tipo           severidad        prioridad  días_resolución  \
2     Problema  0 - Sin Clasificar  0 - Sin Asignar              3.0   
3    Solicitud  0 - Sin Clasificar  0 - Sin Asignar              9.0   
156   Problema          2 - Normal  0 - Sin Asignar              1.0   
157   Problema          2 - Normal  0 - Sin Asignar              1.0   
158   Problema          2 - Normal  0 - Sin Asignar              2.0   

     satisfacción               web  índice_  
2             5.0  www.tractchun.es      1.0  
3             5.0  www.tractchun.es      1.0  
156           5.0  www.tractchun.es      1.0  
157           5.0  www.tractchun.es      1.0  
158           5.0  www.tractchun.es      1.0

3. ¿Y para la categorías de "Hardware" y "Software"? Cread un dataframe nuevo con los resultados de este filtrado de datos. 📌 Nota Este ejercicio se puede resolver usando dos condiciones de filtrado o usando el método isin(). Hacedlo con ambas herramientas.

In [9]:
# metodo isin
df_hs = df[df['categoría'].isin(['Hardware','Software'])]
df_hs.sample(5)

id_ticket       fecha  id_empleado  id_agente categoría  \
7390   KWLTER-5942612616  2016-08-30        181.0        6.0  Hardware   
4145   KDLTER-5042680378  2016-11-06       1267.0       13.0  Hardware   
10806  TDLTER-7042546326  2016-06-25        472.0       23.0  Software   
4082   KDLTER-1642541671  2016-06-20        777.0       46.0  Software   
1423   GHLTER-0242727327  2016-12-23        672.0       33.0  Software   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
7390   Solicitud  2 - Normal         1 - Baja              1.0           3.0   
4145   Solicitud  2 - Normal  0 - Sin Asignar             15.0           3.0   
10806  Solicitud  2 - Normal  0 - Sin Asignar             10.0           5.0   
4082   Solicitud  2 - Normal  0 - Sin Asignar              6.0           5.0   
1423   Solicitud  2 - Normal         3 - Alta              2.0           5.0   

                    web  índice_  
7390   www.tractchun.es      1.0  
4145   www.tractchun.es      1.0  
10806  www.tractchun.es      1.0  
4082   www.tractchun.es      1.0  
1423   www.tractchun.es      1.0

In [10]:
cond1 = df['categoría'] == 'Software'
cond2 = df['categoría'] == 'Hardware'
df_hs2 = df[(cond1 | cond2)]
df_hs2.sample(5)

id_ticket       fecha  id_empleado  id_agente categoría  \
10207  SWLTER-0642471013  2016-04-11        101.0       50.0  Hardware   
2421   GHREET-4642418555  2016-02-18       1295.0       35.0  Software   
8941   SHLTET-0042558832  2016-07-07       1003.0       28.0  Hardware   
3340   GWLTER-6742687861  2016-11-13        826.0       18.0  Hardware   
10783  TDLTER-5342445648  2016-03-16        434.0       26.0  Software   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
10207  Solicitud  2 - Normal         1 - Baja             13.0           4.0   
2421    Problema   1 - Menor         3 - Alta              0.0           5.0   
8941   Solicitud  2 - Normal         3 - Alta              7.0           1.0   
3340   Solicitud  2 - Normal         1 - Baja              6.0           1.0   
10783  Solicitud  2 - Normal  0 - Sin Asignar              3.0           4.0   

                    web  índice_  
10207  www.tractchun.es      1.0  
2421   www.tractchun.es      1.0  
8941   www.tractchun.es      1.0  
3340   www.tractchun.es      1.0  
10783  www.tractchun.es      1.0

In [11]:
cond1 = df['categoría'] == 'Software'
cond2 = df['categoría'] == 'Hardware'
df_hs2 = df[(df['categoría'] == 'Sistema') | (df['categoría'] == 'Hardware')]
df_hs2.sample(5)

id_ticket       fecha  id_empleado  id_agente categoría  \
12035  THLTSR-7842541177  2016-06-20        927.0       31.0   Sistema   
2225   GHLTSR-8642598631  2016-08-16       1453.0        6.0   Sistema   
6268   KHLTST-1642704038  2016-11-30        113.0       10.0   Sistema   
12492  TMLTSR-1542490455  2016-04-30       1465.0       24.0   Sistema   
5249   KHLESR-1542693319  2016-11-19        351.0        3.0   Sistema   

            tipo   severidad  prioridad  días_resolución  satisfacción  \
12035  Solicitud  2 - Normal   3 - Alta              8.0           4.0   
2225   Solicitud  2 - Normal   3 - Alta              5.0           5.0   
6268   Solicitud  2 - Normal   3 - Alta              7.0           1.0   
12492  Solicitud  2 - Normal  2 - Media              5.0           4.0   
5249    Problema  2 - Normal   3 - Alta              5.0           3.0   

                    web  índice_  
12035  www.tractchun.es      1.0  
2225   www.tractchun.es      1.0  
6268   www.tractchun.es      1.0  
12492  www.tractchun.es      1.0  
5249   www.tractchun.es      1.0

- 3.1 De los datos filtrados en el punto 3. ¿Cuántas de ellas son de tipo "Solicitud" y cuántas de tipo "Problema"?

In [12]:
df_hs['categoría'].value_counts()

Software    2576
Hardware    1254
Name: categoría, dtype: int64

- 3.2 ¿Cuántas solicitudes tienen una satisfacción de más de 3?

In [13]:
df_hssatis = df_hs[df_hs['satisfacción'] > 3]
df_hssatis.shape
# hay 2995

(2995, 12)

- 3.3 ¿Y cuántas tienen una satisfacción mayor o igual que 4 y han tardado menos de 3 días de resolución?

In [14]:
cond_a = df_hs['satisfacción'] >= 4
cond_b = df_hs['días_resolución'] < 3

In [15]:
df_satisresol = df_hs[(cond_a & cond_b)]
df_satisresol.sample(5)

id_ticket       fecha  id_empleado  id_agente categoría  \
5662  KHLTET-9242385794  2016-01-16        939.0       17.0  Software   
1462  GHLTER-2242706586  2016-12-02        498.0        5.0  Software   
1164  GHLEET-0042676963  2016-11-02        556.0       29.0  Software   
3789  KDLEER-2142370531  2016-01-01        533.0       45.0  Software   
8613  SHLEER-3042588973  2016-08-06       1017.0       49.0  Software   

           tipo   severidad        prioridad  días_resolución  satisfacción  \
5662  Solicitud  2 - Normal         3 - Alta              0.0           5.0   
1462  Solicitud  2 - Normal         3 - Alta              2.0           5.0   
1164   Problema  2 - Normal         3 - Alta              0.0           5.0   
3789   Problema  2 - Normal  0 - Sin Asignar              1.0           4.0   
8613   Problema  2 - Normal         3 - Alta              0.0           4.0   

                   web  índice_  
5662  www.tractchun.es      1.0  
1462  www.tractchun.es      1.0  
1164  www.tractchun.es      1.0  
3789  www.tractchun.es      1.0  
8613  www.tractchun.es      1.0

In [16]:
df_satisresol.shape
# hay 574

(574, 12)

4 - Ahora nuestro jefe quiere saber que cantidad de tickets ha habido en la segunda quincena de Enero. Cread un DataFrame nuevo con los tickets correspondientes a la segunda quincena de Enero.

In [17]:
df_hs

id_ticket       fecha  id_empleado  id_agente categoría  \
4      GDLEER-0042524120  2016-06-03       1842.0       31.0  Software   
5      GDLEER-0142608095  2016-08-26         59.0       20.0  Software   
6      GDLEER-0242564650  2016-07-13       1175.0       36.0  Software   
7      GDLEER-0542574815  2016-07-23        561.0       18.0  Software   
8      GDLEER-0842457219  2016-03-28         71.0       12.0  Software   
...                  ...         ...          ...        ...       ...   
13034  TWRTER-2042528927  2016-06-07        462.0       19.0  Software   
13035  TWRTER-5042710887  2016-12-06       1098.0       38.0  Hardware   
13036  TWRTER-8742418826  2016-02-18        942.0       28.0  Software   
13037  TWRTER-9542402645  2016-02-02        334.0       26.0  Software   
13038  TWRTER-9642514691  2016-05-24        909.0       16.0  Hardware   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
4       Problema  2 - Normal  0 - Sin Asignar              0.0           5.0   
5       Problema  2 - Normal  0 - Sin Asignar              1.0           1.0   
6       Problema  2 - Normal  0 - Sin Asignar              2.0           1.0   
7       Problema  2 - Normal  0 - Sin Asignar              5.0           5.0   
8       Problema  2 - Normal  0 - Sin Asignar              8.0           5.0   
...          ...         ...              ...              ...           ...   
13034  Solicitud   1 - Menor         1 - Baja             10.0           4.0   
13035  Solicitud   1 - Menor         1 - Baja             10.0           4.0   
13036  Solicitud   1 - Menor         1 - Baja              7.0           5.0   
13037  Solicitud   1 - Menor         1 - Baja              5.0           5.0   
13038  Solicitud   3 - Mayor         1 - Baja              7.0           5.0   

                    web  índice_  
4      www.tractchun.es      1.0  
5      www.tractchun.es      1.0  
6      www.tractchun.es      1.0  
7      www.tractchun.es      1.0  
8      www.tractchun.es      1.0  
...                 ...      ...  
13034  www.tractchun.es      1.0  
13035  www.tractchun.es      1.0  
13036  www.tractchun.es      1.0  
13037  www.tractchun.es      1.0  
13038  www.tractchun.es      1.0  

[3830 rows x 12 columns]

In [18]:
patron = '2016-01-(1[5-9]|[2-3][0-9])'

In [19]:
df_enero = df_hs[df_hs['fecha'].str.contains(patron)]
df_enero.sample(5)

/tmp/ipykernel_839/510028909.py:1: UserWarning: This pattern has match groups. To actually get the groups, use str.extract.
  df_enero = df_hs[df_hs['fecha'].str.contains(patron)]


id_ticket       fecha  id_empleado  id_agente categoría  \
10870  TDLTET-7342384993  2016-01-15       1889.0        9.0  Software   
7367   KWLTER-4542392099  2016-01-23       1759.0       40.0  Software   
5496   KHLTER-6142392347  2016-01-23        544.0       13.0  Software   
1713   GHLTET-9242386491  2016-01-17         39.0       24.0  Software   
254    GDLTER-0542396537  2016-01-27       1373.0        5.0  Software   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
10870  Solicitud  2 - Normal  0 - Sin Asignar              3.0           5.0   
7367   Solicitud  2 - Normal         1 - Baja              5.0           5.0   
5496   Solicitud  2 - Normal         3 - Alta              9.0           3.0   
1713   Solicitud  2 - Normal         3 - Alta              2.0           5.0   
254    Solicitud  2 - Normal  0 - Sin Asignar              5.0           5.0   

                    web  índice_  
10870  www.tractchun.es      1.0  
7367   www.tractchun.es      1.0  
5496   www.tractchun.es      1.0  
1713   www.tractchun.es      1.0  
254    www.tractchun.es      1.0

- 4.1 ¿Ha habido algun ticket en esta quincena que corresponda a la categoría de Solicitud?

In [20]:
df_enero[df_enero['tipo'] == 'Solicitud'].shape
# 155

(155, 12)

- 4.2 Revisando los datos, nuestro jefe se ha dado cuenta que hay muchos tickets no tienen ninguna prioridad asignada. Esto no debería seguir repitiéndose ya que cada ticket debe tener una prioridad para saber que urgencia debemos darle a cada ticket. 

- Usando el **str.contains** extraed los tickets que no tengan una prioridad asignada o que tengan una prioridad baja.

In [21]:
patron2 = '^[0]|^[1]'

In [22]:
df_prioridad = df[df['prioridad'].str.contains(patron2)]

In [23]:
df_prioridad.sample(5)

id_ticket       fecha  id_empleado  id_agente      categoría  \
3634   GWLTSR-6542585576  2016-08-03        337.0       45.0        Sistema   
10530  SWRENR-2042566810  2016-07-15        281.0       38.0  Acceso&%Login   
10870  TDLTET-7342384993  2016-01-15       1889.0        9.0       Software   
4996   KDRTNT-2042381876  2016-01-12         87.0       18.0  Acceso&%Login   
3772   KDDESR-4642712994  2016-12-08       1369.0       19.0        Sistema   

            tipo           severidad        prioridad  días_resolución  \
3634   Solicitud          2 - Normal         1 - Baja              5.0   
10530   Problema           3 - Mayor         1 - Baja              0.0   
10870  Solicitud          2 - Normal  0 - Sin Asignar              3.0   
4996   Solicitud           3 - Mayor  0 - Sin Asignar              0.0   
3772    Problema  0 - Sin Clasificar  0 - Sin Asignar              6.0   

       satisfacción               web  índice_  
3634            1.0  www.tractchun.es      1.0  
10530           5.0  www.tractchun.es      1.0  
10870           5.0  www.tractchun.es      1.0  
4996            3.0  www.tractchun.es      1.0  
3772            4.0  www.tractchun.es      1.0

5 - Nuestro jefe quiere seguir buscando patrones y quiere saber si existe una relación entre el número de días que tarde en cerrarse un ticket y la satisfacción de los clientes. Para eso nos pide que:

- 5.1 Extraigamos los tickets en lo que pasaron 5 o más días en ser solucionados.

In [24]:
df_resolucion_larga = df[df['días_resolución'] >= 5]

In [25]:
df_resolucion_larga.sample(5)

id_ticket       fecha  id_empleado  id_agente categoría  \
8955   SHLTET-2142509112  2016-05-19       1101.0        1.0  Software   
6960   KMLTSR-1742492328  2016-05-02       1572.0       23.0   Sistema   
10468  SWLTSR-6542445611  2016-03-16       1281.0       46.0   Sistema   
4913   KDLTST-6642526676  2016-06-05       1257.0       16.0   Sistema   
6269   KHLTST-1742613237  2016-08-31        923.0        2.0   Sistema   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
8955   Solicitud  2 - Normal         3 - Alta             10.0           4.0   
6960   Solicitud  2 - Normal        2 - Media              7.0           5.0   
10468  Solicitud  2 - Normal         1 - Baja             15.0           5.0   
4913   Solicitud  2 - Normal  0 - Sin Asignar              6.0           3.0   
6269   Solicitud  2 - Normal         3 - Alta              7.0           3.0   

                    web  índice_  
8955   www.tractchun.es      1.0  
6960   www.tractchun.es      1.0  
10468  www.tractchun.es      1.0  
4913   www.tractchun.es      1.0  
6269   www.tractchun.es      1.0

- 5.2 Extraigamos los tickets en los que pasaron menos de 5 días en ser solucionados.

In [26]:
df_resolucion_corta = df[df['días_resolución'] < 5]

In [27]:
df_resolucion_corta.sample(5)

id_ticket       fecha  id_empleado  id_agente      categoría  \
10536  SWRESR-1242729817  2016-12-25        611.0        8.0        Sistema   
12192  THRTNT-2142428985  2016-02-28       1318.0       39.0  Acceso&%Login   
5067   KHLEER-4342718449  2016-12-14       1384.0       44.0       Software   
10379  SWLTNR-9042626718  2016-09-13        591.0        7.0  Acceso&%Login   
6806   KMLTNR-1042636327  2016-09-23        362.0       43.0  Acceso&%Login   

            tipo   severidad  prioridad  días_resolución  satisfacción  \
10536   Problema   1 - Menor   1 - Baja              2.0           5.0   
12192  Solicitud   3 - Mayor   3 - Alta              1.0           4.0   
5067    Problema  2 - Normal   3 - Alta              3.0           4.0   
10379  Solicitud  2 - Normal   1 - Baja              0.0           4.0   
6806   Solicitud  2 - Normal  2 - Media              0.0           1.0   

                    web  índice_  
10536  www.tractchun.es      1.0  
12192  www.tractchun.es      1.0  
5067   www.tractchun.es      1.0  
10379  www.tractchun.es      1.0  
6806   www.tractchun.es      1.0

- 5.3 Usando como referencia los dos conjuntos de datos creados en los dos pasos anteriores:

- ¿Qué valores tenemos de satisfacción en cada uno de ellos?

In [28]:
df_resolucion_larga['satisfacción'].value_counts()

5.0    2873
4.0    1803
1.0     718
3.0     533
2.0     109
Name: satisfacción, dtype: int64

In [29]:
df_resolucion_corta['satisfacción'].value_counts()

5.0    3314
4.0    1926
1.0     865
3.0     593
2.0     167
Name: satisfacción, dtype: int64

- ¿En cuál de ellos hay más puntuaciones de 5 en relación a la satisfacción?

En la tabla en la que los dias de resolucion es más corta hay más puntuaciones de satisfaccion 5 (3314)
Frente a la tabla en la que se encuentran los tikets con un tiempo de resolucion más largo (2873)

6 - Revisando los dataframes generados en el ejercicio anterior, nos hemos dado cuenta que hay una serie de empleados que tardan demasiado en resolver los problemas planteados por los clientes. Nuestro jefe quiere extraer los datos de estos empleados en concreto para saber que esta pasando. 

- En concreto estos empleados son aquellos que tienen los siguientes id's: 1118.0, 730.0, 97.0, 1745.0. Crea un dataframe donde solo tengamos la información de estos empleados. Usad el método isin(), para resolver este ejercicio.

In [30]:
df_empleadoslentos = df[df['id_empleado'].isin([1118.0, 730.0, 97.0, 1745.0])]

In [31]:
df_empleadoslentos.sample(3)

id_ticket       fecha  id_empleado  id_agente categoría  \
12980  TWLTSR-6442469687  2016-04-09       1118.0       26.0   Sistema   
8435   SDLTSR-5642730382  2016-12-26       1118.0       43.0   Sistema   
7090   KMLTSR-9542558851  2016-07-07       1745.0       48.0   Sistema   

            tipo   severidad        prioridad  días_resolución  satisfacción  \
12980  Solicitud  2 - Normal         1 - Baja             14.0           4.0   
8435   Solicitud  2 - Normal  0 - Sin Asignar              6.0           1.0   
7090   Solicitud  2 - Normal        2 - Media              5.0           5.0   

                    web  índice_  
12980  www.tractchun.es      1.0  
8435   www.tractchun.es      1.0  
7090   www.tractchun.es      1.0

7 - Utilizando la columna de fecha cread una nueva columna llamada semestre que contenga 1er semestre o 2º semestre. 
- Aquellos tickets que pertenezcan a los seis primeros meses tendran el valor de "1er trismestre" y los tickets de julio en adelante tendrán el valor de "2º semestre".

In [32]:
df.sample(3)

id_ticket       fecha  id_empleado  id_agente      categoría  \
3514   GWLTNR-8142613438  2016-08-31       1553.0       44.0  Acceso&%Login   
1894   GHLTNR-9042729430  2016-12-25        883.0       34.0  Acceso&%Login   
12000  THLTSR-5742413281  2016-02-13       1168.0       32.0        Sistema   

            tipo   severidad prioridad  días_resolución  satisfacción  \
3514   Solicitud  2 - Normal  1 - Baja              1.0           5.0   
1894   Solicitud  2 - Normal  3 - Alta              0.0           5.0   
12000  Solicitud  2 - Normal  3 - Alta              7.0           5.0   

                    web  índice_  
3514   www.tractchun.es      1.0  
1894   www.tractchun.es      1.0  
12000  www.tractchun.es      1.0

In [33]:
p_semestre1 = '2016-0[1-6].*'           
p_semestre2 = '2016-0[7-9]|1[0-2].*' 

In [34]:
condicion1 = df['fecha'].str.contains('2016-0[1-6].*')
condicion2 = df['fecha'].str.contains('2016-0[7-9]|1[0-2].*')

In [35]:
opciones = ['1º semestre', '2º semestre']

In [36]:
df['semestre'] = np.select([condicion1,condicion2], opciones)

In [37]:
df.sample(3)

id_ticket       fecha  id_empleado  id_agente      categoría  \
1957   GHLTNT-5042700235  2016-11-26       1073.0       42.0  Acceso&%Login   
10385  SWLTNR-9142685075  2016-11-11        337.0       50.0  Acceso&%Login   
10401  SWLTSR-1442616651  2016-09-03       1615.0        6.0        Sistema   

            tipo   severidad prioridad  días_resolución  satisfacción  \
1957   Solicitud  2 - Normal  3 - Alta              0.0           5.0   
10385  Solicitud  2 - Normal  1 - Baja              1.0           5.0   
10401  Solicitud  2 - Normal  1 - Baja              5.0           4.0   

                    web  índice_     semestre  
1957   www.tractchun.es      1.0  2º semestre  
10385  www.tractchun.es      1.0  2º semestre  
10401  www.tractchun.es      1.0  2º semestre

8 - Guardad como un archivo csv nuevo. Lo usaremos en el pair programming de mañana.

In [38]:
df.to_csv("../datos/df_tikets.csv")

FileNotFoundError: [Errno 2] No such file or directory: 'datos/df_tikets.csv'